Extension/src/LanguageServer/client.ts

```diff
@@ -686,7 +686,10 @@ class DefaultClient implements Client {
             if (showIntelliSenseFallbackMessage.Value) {
                 let learnMorePanel: string = "Learn More";
                 let dontShowAgain: string = "Don't Show Again";
-                vscode.window.showInformationMessage("Configure includePath for better IntelliSense results.", learnMorePanel, dontShowAgain).then((value) => {
+                let fallbackMsg: string = this.configuration.VcpkgInstalled ?   
+                    "Update your IntelliSense settings or use vcpkg to install libraries to help find missing headers." :
```

Comment:
double check if there is an capitalization for `vcpkg` and then you're good to go.